diff --git a/Makefile.in b/Makefile.in
index 1597d39..0413d9d 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -1773,7 +1773,7 @@ EXTRA_DIST = README-dev ChangeLog.O m4/gnulib-cache.m4 man/hello.x \
 # Subdirectories to descend into.
 SUBDIRS = po
 hello_SOURCES = src/hello.c src/system.h
-hello_LDADD = $(LIBINTL) $(top_builddir)/lib/lib$(PACKAGE).a
+hello_LDADD = $(LIBINTL) $(LIBICONV) $(top_builddir)/lib/lib$(PACKAGE).a
 CLEANFILES = $(am__append_2) lib/configmake.h lib/configmake.h-t
 MAINTAINERCLEANFILES = lib/iconv_open-aix.h lib/iconv_open-hpux.h \
 	lib/iconv_open-irix.h lib/iconv_open-osf.h \
